metrics-operator/chart/values.yaml

```diff
@@ -93,6 +93,10 @@ annotations: {}
 podAnnotations: {}
 ## @param     kubernetesClusterDomain overrides cluster.local
 kubernetesClusterDomain: cluster.local
+## @extra   customMetricsAPIService
```

Comment:
Let's remove this extra, it's not needed here
```suggestion
```

Reply:
Alright